components/sections/ExportTranslations.tsx

```diff
@@ -75,17 +72,28 @@ const ExportTranslations = ({ translations, categoryName, triggerLabel }: Export
 
   const translSep = translationSeparator === TRANS_SEP_CUSTOM ? customTranslationSeparator : translationSeparator;
   const phraseSep = phraseSeparator === PHRASE_SEP_CUSTOM ? customPhraseSeparator : phraseSeparator;
-  const phrases = translations
-    .map(
-      (translation) =>
-        translation.getTranslation(currentLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(currentLanguage)}]` : '') +
-        translSep +
-        translation.getTranslation(otherLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(otherLanguage)}]` : '') +
-        phraseSep
-    )
-    .map((translation) => unescapeTabsAndNewlines(translation));
+  const source =
+    categoryName != 'všechny fráze'
+      ? translations.categories.filter((c) => `${c.name.main} - ${c.name.source}` === categoryName)
+      : translations.categories;
+  const phrases = source.map((translation) => {
+    const text = '[' + translation.name.main + ']' + translSep + '[' + translation.name.source + ']' + '\n';
+    return text.concat(
+      translation.phrases
+        .map((phraseId) => {
+          const phrase = getPhraseById(translations, phraseId);
+          return (
+            phrase.getTranslation(currentLanguage) +
+            (includeTranscriptions ? ` [${phrase.getTranscription(currentLanguage)}]` : '') +
+            translSep +
+            phrase.getTranslation(otherLanguage) +
+            (includeTranscriptions ? ` [${phrase.getTranscription(otherLanguage)}]` : '') +
+            phraseSep
+          );
+        })
+        .join('')
+    );
+  });
```

Comment:
@met  Separating categories with '\n' make it more readable, if it is wanted. Formatting has to be made in more places, for clipboard and for export to TXT. Consider to create formatting function.